torch_frame/data/dataset.py

```diff
@@ -128,14 +136,17 @@ def __call__(
         r"""Convert a given :obj:`DataFrame` object into :class:`TensorFrame`
         object."""
 
-        xs_dict: Dict[torch_frame.stype, List[Tensor]] = defaultdict(list)
+        xs_dict: Dict[torch_frame.stype,
+                      List[Union[Tensor,
+                                 MultiNestedTensor]]] = defaultdict(list)
```

Comment:
Can we group this as `TensorData = Union[Tensor, MultiNestedTensor]` (or similar)  in `typing.py`.

Reply:
Sure